src/OrchardCore.Modules/OrchardCore.ArchiveLater/Handlers/ArchiveLaterPartHandler.cs

```diff
@@ -0,0 +1,18 @@
+using System.Threading.Tasks;
+using OrchardCore.ArchiveLater.Models;
+using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement.Handlers;
+
+namespace OrchardCore.ArchiveLater.Handlers;
+
+// TODO: Rmove this one the PR is ready
+public class ArchiveLaterPartHandler : ContentPartHandler<ArchiveLaterPart>
+{
+    public override Task UnpublishedAsync(PublishContentContext context, ArchiveLaterPart part)
+    {
+        part.ScheduledArchiveUtc = null;
+        part.Apply();
```

Comment:
If we agree to do it in the bgTask itself, we could remove this handler.

See my comment in the main thread.

Reply:
Yes I think removed in the other PR if I'm not wrong